marklogic-data-hub-central/ui/src/pages/noMatchRedirect.tsx

```diff
@@ -14,16 +14,16 @@ const NoMatchRedirect = ({history}) => {
   }, []);
 
   const backToHomePage = () => {
-    return user.authenticated ? history.push('/home') : history.push('/');
+    return user.authenticated ? history.push('/tiles') : history.push('/');
```

Comment:
Fixed this redirect on 404 to /tiles from /home for "Back Home" button

Reply:
Thanks for fixing this.